### DISCUSSION TITLE
Adding a beautiful error page and error tracking service for Magento

### DISCUSSION THREAD

**Discussion by miljkovic5** [wardenenv/warden#795] [Ideas]
Can we add this module as dev dependency when we install Magento project? https://github.com/swissup/module-ignition or https://github.com/yireo/Yireo_Whoops

**Comment by bap14** [comment]
Warden is focused more on the infrastructure side of things than the software running on it. To that end, given that composer.json and composer.lock should be committed to the repository, and thus are per-project, I would say this is something that each project / developer needs to decide they want to use, not something we just randomly add when installing Magento.
